Vagrantfile

```diff
@@ -0,0 +1,35 @@
+Vagrant.configure("2") do |config|
+  config.vm.box = "hashicorp/bionic64"
+  config.vm.provision "shell", privileged: false, inline: <<-SHELL
+    sudo apt-get update
+
+    #install pyenv prereqs
+    sudo apt-get install -y build-essential libssl-dev zlib1g-dev libbz2-dev \
+    libreadline-dev libsqlite3-dev wget curl llvm libncurses5-dev libncursesw5-dev \
+    xz-utils tk-dev libffi-dev liblzma-dev python-openssl git
+
+    #install pyenv
+    git clone https://github.com/pyenv/pyenv.git ~/.pyenv
+    echo 'export PYENV_ROOT="$HOME/.pyenv"' >> ~/.bash_profile
```

Comment:
Not vital, but worth noting that it's generally better to use `.profile` over `.bash_profile` - see [this stack overflow answer](https://superuser.com/questions/789448/choosing-between-bashrc-profile-bash-profile-etc) for a quick summary